CHANGELOG.md

```diff
@@ -44,6 +44,13 @@ Given a version number `MAJOR.MINOR.PATCH`, we increment the:
   - `terramate debug show generate-origins`
   - `terramate debug show runtime-env`
 - Improvements in the output of `list`, `run` and `create` commands.
+- Promote cloud commands
```

Comment:
This one must be moved to the 0.4.5 section now.

Reply:
fixed